src/main/java/seedu/address/logic/commands/ShowEventsCommand.java

```diff
@@ -0,0 +1,22 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.model.Model;
+
+/**
+ * Lists all events in the address book to the user.
+ */
+public class ShowEventsCommand extends Command {
```

Comment:
Might want to override `toString` and `equals` methods for downstream usage.

Reply:
Added!